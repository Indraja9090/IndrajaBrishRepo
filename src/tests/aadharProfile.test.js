// AadharProfile.test.js
import React from 'react';
import renderer from 'react-test-renderer';
import AadharProfile from '../components/aadharProfile';

describe('AadharProfile Snapshot', () => {
  it('renders correctly with initial props', () => {
    const tree = renderer.create(
        <AadharProfile
          name="Indraja"
          aadharNumber="1234-5678-9012"
          dob="01-01-1990"
          gender="Female"
        />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});