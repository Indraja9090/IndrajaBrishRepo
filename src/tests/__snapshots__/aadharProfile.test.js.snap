// Jest Snapshot v1, https://goo.gl/fbAQLP
// aadharProfile.test.js.snap
exports[`AadharProfile Snapshot renders correctly with initial props 1`] = `null`;
